### DISCUSSION TITLE
Vercel cant find images stored on the public root directory folder

### DISCUSSION THREAD

**Discussion by aivaneezy1** [vercel/community#7449] [Feature Question] [answered]
### Question

I need some help.Images are not showing on deployed website, they are stored in the public root directory. I already tripled check the path and its 100% correct es: "src="/images/filename.jpg" im using relative path. when the project ran locally images are  showing only on vercel they are not showing at all.Does anyone faced this same problem before? if yes, how did you fixed it?

**Comment by pawlean** [comment]
Hi @aivaneezy1!

Have you tried configuring your `vercel.json` with something like:

```
{
  "rewrites": [{ "source": "/(.*)", "destination": "/index.html" }]
}
```
See [relevant documentation](https://vercel.com/docs/edge-network/rewrites).

**Reply by aivaneezy1** [reply, marked as answer]
Hello, sorry for the late reply. where can i find the vercel.json because on my project i dont have one.  Should i just create one in the root of my project?.
![image](https://github.com/user-attachments/assets/6e0ae795-9426-44c1-a5da-ccaaef03ee0c)

**Reply by pawlean** [reply]
Yes! 😄

[a comment by an automated account is not shown]
